support/Pipelines/Bbh/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+import click
+
+from .InitialData import generate_id_command
+from .Inspiral import start_inspiral_command
+from .Ringdown import start_ringdown_command
+
+
+@click.group(name="bbh")
+def bbh_pipeline():
+    """Pipeline for binary black hole simulations."""
+    pass
+
+
+bbh_pipeline.add_command(generate_id_command)
+bbh_pipeline.add_command(start_inspiral_command)
+bbh_pipeline.add_command(start_ringdown_command)
+
+if __name__ == "__main__":
+    bbh_pipeline(help_option_names=["-h", "--help"])
```

Comment:
Maybe print a pretty click warning somewhere that this is still very experimental and shouldn't be used for actual runs yet?

Reply:
It's not more experimental than the BBH input file that you would copy from tests/InputFiles :D

Reply:
It's not the input file that's experimental, it's the pipeline. My idea of a pipeline is smooth seamless transitions with little input required from the user. At the moment, this isn't that. But I hope it will be!

Reply:
Ah ok, agreed! Added some warnings.